Library/Formula/git-subrepo.rb

```diff
@@ -4,15 +4,8 @@ class GitSubrepo < Formula
   head "https://github.com/ingydotnet/git-subrepo.git"
 
   stable do
```

Comment:
Without the patch, we can do without the stable block; `url` and `sha256` can be placed above `head` now.